galaxy_ng/tests/integration/constants.py

```diff
@@ -251,3 +251,36 @@
         "token": "abcdefghijklmnopqrstuvwxyz1234567895",
     },
 }
+
+BETA_GALAXY_PROFILES = {
```

Comment:
Another place where the context is stage only, so i'd prefer a var like BETA_GALAXY_STAGE_PROFILES